### PR TITLE
Create contact -form-section 

### PR DESCRIPTION
**Pull request by mandelatuks**
# the following were added:

- Add a contact form field area 
- Add submit button to the form 
- Add social icons to the form
- Add email to the form
- No linters error

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -385,13 +385,101 @@ h3 {
...
-  width: 90%;
-  margin: auto;
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  top: 485px;
+  margin-top: 15px;
+}
+
```
Comment (AmaduKamara) [inline styles.css:395]: You are doing great so far but there are a few changes that you've left out to complete your design

[ Required Changes ]

- [ ] Please add the background image for the contact section as it's in the Figma design. It's highlighted in a green arrow in the comment photo below. Please fix that 👍 
- [ ] Kindly add the remaining social media icons that are in the Figma design.
- Spoiler Alert: You will be creating an account for all those social media icons as you go in the course program and hence, it's good you have them now in your portfolio app as you'll update their links once you create the accounts. 👍
Your Design
image

The Figma Design
image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @mandelatuks and @Lembani 

Good job so far! 💪
There are some issues that you still need to work on to go to the next project but you are almost there!

AlonzoLeroneThisCloseGIF

## Highlights
✔️ Good README
✔️ No Linter errors
✔️ Contact form is working fine
✔️ Good commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mandelatuks** [comment]
@AmaduKamara 
the image was showing on my side but i hope now you can see it

Screenshot 2022-03-23 015540

**Review by keroloussamy — APPROVED** [review]
Hi @mandelatuks, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**